Src/FluentAssertions/Numeric/SByteAssertions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Diagnostics;
+
+namespace FluentAssertions.Numeric
+{
+    /// <summary>
+    /// Contains a number of methods to assert that a <see cref="sbyte"/> is in the expected state.
```

Comment:
With the assumption that "sbyte" is pronounced "esbyte"
```suggestion
    /// Contains a number of methods to assert that an <see cref="sbyte"/> is in the expected state.
```